tests/unit/test_apm_config/test_apm_config.py

```diff
@@ -673,6 +673,102 @@ def test_set_config_value_default_log_type(
         if old_log_type:
             os.environ["SW_APM_LOG_TYPE"] = old_log_type
 
+    def test_set_config_value_default_export_logs_enabled(
+        self,
+    ):
+        test_config = apm_config.SolarWindsApmConfig()
+        assert test_config.get("export_logs_enabled") == False
+
+    def test_set_config_value_ignore_export_logs_enabled(
+        self,
+        caplog,
+        setup_caplog,
+        mock_env_vars,
+    ):
+        # Save any export_logs_enabled in os for later
+        old_expt_logs = os.environ.get("SW_APM_EXPORT_LOGS_ENABLED", None)
+        if old_expt_logs:
+            del os.environ["SW_APM_EXPORT_LOGS_ENABLED"]
```

Comment:
Does the Python test runner have before/after hooks that could be used to clear and restore any env vars with SW or OTEL prefixes ?

Reply:
It does, and I will work on that!

Reply:
Added before/after hook in [a4aaf81](https://github.com/solarwinds/apm-python/pull/393/commits/a4aaf811f9ac17315310e95a09b4ca1d831c0615) for test_apm_config and test_distro.